shotover-proxy/tests/transforms/tee.rs

```diff
@@ -0,0 +1,146 @@
+use crate::helpers::ShotoverManager;
+use serial_test::serial;
+use test_helpers::docker_compose::DockerCompose;
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_ignore_matches() {
+    let shotover_manager =
+        ShotoverManager::from_topology_file("tests/test-topologies/tee/ignore.yaml");
+
+    let mut connection = shotover_manager.redis_connection_async(6379).await;
+
+    redis::cmd("SET")
```

Comment:
I think we should run the same command for every test.
This SET seems fine, lets run just this one command for every test, having the tests consistent makes it easier to ensure they are all correct and makes it easier to investigate failures.